source/index.txt

```diff
@@ -1,80 +1,247 @@
+:template: product-landing
+:hidefeedback: header
+:noprevnext:
+
 .. program:: mongosh
 
 .. binary:: mongosh
 
 .. _mdb-shell-overview:
 
-===========================
-MongoDB Shell (``mongosh``)
-===========================
+======================================
+Welcome to MongoDB Shell (``mongosh``)
+======================================
+
+.. introduction::
+
+   The |mdb-shell|, ``mongosh``, is a JavaScript and Node.js 16.x |repl| 
+   environment for interacting with MongoDB deployments in Atlas, locally,
+   or on another remote host. Use the |mdb-shell| to test queries and 
+   operations directly with your database.
+
+   .. button:: Download mongosh
+      :uri: https://www.mongodb.com/try/download/shell?jmp=docs
+
+.. image:: /images/mongosh-plp/Technical_SOFTWARE_Terminal(1)_Spot.png
+   :alt: Abstract illustration showing a terminal interacting with different systems 
+
+.. kicker:: What You Can Do
+
+Access MongoDB From Your Shell
+------------------------------
+
+.. tabs::
+
+   .. tab:: Connect to MongoDB
+      :tabid: connect
+
+      .. procedure::
```

Comment:
I agree that the numbered procedure is a little odd here given that a user only wants to do one of these, but the styling with this product landing page template is a bit constrained. IIRC the only way to include text and image side-by-side are by using a procedure like this, or by using a `.. container::` reStructuredText directive, which technically works - we do it in the [Realm SDK docs](https://www.mongodb.com/docs/realm/sdk/swift/) on the SwiftUI/App Services tabs - but the directive is deprecated and causes build warnings.

If I switch to an Atlas-focused procedure instead of just mentioning Atlas as the first connection "option," we lose the prominent info on the landing page that you can _also_ use `mongosh` to connect to local deployments and other remote hosts. I can mention it in the text, but that would likely be missed by people who are just scanning headings to see what you can do.

With that context in mind, would you both still prefer I switch this to a procedure focused on Atlas?

(And, alas, @GaurabAryal , the GIF is an existing asset that I was able to get from the Atlas documentation set. I don't have the tools to modify it. If you think it's unclear, would you rather I remove it from this documentation set? I could put a static illustration there, instead.)

Reply:
I've rewritten this tab per your suggestions, @sarahsimpers  - is this what you had in mind?

Reply:
This is exactly what I had in mind, thank you!